main.py

```diff
@@ -102,6 +109,13 @@ def round_to_micros(t_nanos: int, base: int = 1000) -> int:
     """
     return base * round(t_nanos / base)
 
+def extract_pc(points_egovehicle, points, output_sensor_path, timestamp):
```

Comment:
maybe `write_ply()` is a more suitable fn name here?

Do you mind adding type hints as well on the input and return arguments -- ` ) -> None` most likely

Reply:
Done